Sources/Web3Modal/Screens/ChainSwitch/NetworkDetail/NetworkDetailView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+import Web3ModalUI
+
+struct NetworkDetailView: View {
+    @EnvironmentObject var store: Store
+    
+    @ObservedObject var viewModel: NetworkDetailViewModel
+    
+    
+    var body: some View {
+        VStack {
+            content()
+                .onAppear {
```

Comment:
may be .task{ } will be enough here? It also has async context so no need to create extra Tasks

Reply:
but .task is executing on first view load only

Reply:
Yeah but it is also iOS 15+